.circleci/config.yml

```diff
@@ -148,6 +148,27 @@ jobs:
                   working_directory: << parameters.working_directory >>
             - notify-on-failure
 
+    validate:
+        docker:
+            - image: cimg/openjdk:11.0
+        resource_class: small
+        steps:
+            - checkout
+            - attach_workspace:
+                  at: .
+            - restore-maven-cache
+            - run:
+                  name: "validate project"
+                  command: |
+                      mvn -pl '!gravitee-apim-console-webui, !gravitee-apim-portal-webui' -s .artifactory.settings.xml validate --no-transfer-progress -T 2C
```

Comment:
Is there a way to skip this validation in the `build` job?

Reply:
the `-DskipTests` option is used to skip validation. But it could be better since for the test job, we will obviously not skip tests. So the validation will be executed again